tests/h/search/config_test.py

```diff
@@ -178,12 +195,78 @@ def test_updates_index_atomically(self, client):
 
     def test_raises_if_called_for_concrete_index(self, client):
         """Raise if called for a concrete index."""
-        client.conn.indices.get_alias.side_effect = NotFoundError('test', 'test desc')
+        client.conn.indices.get_alias.side_effect = (
+            elasticsearch1.exceptions.NotFoundError('test', 'test desc'))
```

Comment:
I think this needs to be parametrized with the `NotFoundError` classes from the two different versions of the ES library?

Reply:
See my previous comment that I tagged you on. I'm not sure we need to test the same thing twice but if you still feel like it needs to test it again feel free to add it.